tasmota/xdrv_02_mqtt.ino

```diff
@@ -23,9 +23,9 @@
 #define MQTT_WIFI_CLIENT_TIMEOUT   200    // Wifi TCP connection timeout (default is 5000 mSec)
 #endif
 
-#ifdef USE_MQTT_AZURE_IOT
 #include <bearssl\bearssl.h>
 #include <base64.hpp>
+#ifdef USE_MQTT_AZURE_IOT
```

Comment:
Why is this last change used for?

Reply:
I received the failed to build message when they were not global, as the method azurePreSharedKeytoSASToken was global and not under the #ifdef